pom.xml

```diff
@@ -94,6 +94,7 @@
 		<dependency>
 			<groupId>org.springframework.boot</groupId>
 			<artifactId>spring-boot-starter-data-mongodb</artifactId>
+			<version>2.2.5.RELEASE</version>
```

Comment:
Not sure if changes to pom.xml should be included in the pr 
@andrewhlu  what do you think about this?